package.json

```diff
@@ -18,9 +18,9 @@
     "reimport:data": "start-server-and-test 'http-server data/dist -c-1 -p 8001' http://localhost:8001 reimport:data_",
     "reimport:data_": "geoprocessing reimport:data",
     "publish:data": "geoprocessing publish:data",
-    "install:data": "mkdir -p data/bin && cp -r node_modules/@seasketch/geoprocessing/dist/data/bin/* data/bin && mkdir -p data/scripts && cp -r node_modules/@seasketch/geoprocessing/dist/data/scripts/* data/scripts",
+    "install:scripts": "mkdir -p data/bin && cp -r node_modules/@seasketch/geoprocessing/dist/data/bin/* data/bin && mkdir -p scripts && cp -r node_modules/@seasketch/geoprocessing/dist/base-project/scripts/* scripts",
     "translation:install": "rm -rf src/i18n/baseLang && cp -r node_modules/@seasketch/geoprocessing/dist/base-project/src/i18n/baseLang src/i18n",
-    "translation:extract": "rm -rf src/i18n/lang/en && NODE_ENV=development npx babel 'src/**/*.{ts,tsx}' > /dev/null && mv extractedTranslations/en src/i18n/lang/en && rm -rf extractedTranslations && echo 'Translations extracted to src/lang/en'",
+    "translation:extract": "rm -rf src/i18n/lang/en && NODE_ENV=development npx babel 'src/**/*.{ts,tsx}' > /dev/null && mv extractedTranslations/en src/i18n/lang/en && rm -rf extractedTranslations && npx ts-node scripts/extractExtraTerms.ts && echo 'Translations extracted to src/lang/en'",
```

Comment:
run our new script for extracting extra terms

Reply:
Due to the echo I was checking to see if the extra terms were in src/i18n/lang/en. I would update the echo to `Translations extracted to src/i18n/lang/en and src/i18n/extraTerms.json`. Other than that it worked great.